pom.xml

```diff
@@ -47,6 +47,11 @@
     </build>
     
     <dependencies>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>display-url-api</artifactId>
+            <version>0.2</version>
```

Comment:
Shouldn't that be an optional dependency?
Otherwise the plugin won't run without the display-url-api plugin, right?